examples/cache-gatt-discovery.js

```diff
@@ -8,7 +8,13 @@
 var noble = require('../index');
 const fs = require('fs'); 
 
+// the sensor value to scan for, number of bits and factor for displaying it
+const CHANNEL = process.env['CHANNEL'] ? process.env['CHANNEL'] : 'Temperature'
```

Comment:
or just "Unknown"

Reply:
I'd prefer a working example... at least for parts of the universe ;)